lab_5_scraper/scraper.py

```diff
@@ -170,6 +292,14 @@ def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         Args:
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
+        div = article_soup.find('div', class_= 'article_text')
+        text = []
+        if div is None:
+            return None
```

Comment:
why do you explicitly return None?

Reply:
no, this check is good, I do not understand why you changed it. it allowed your code be less nested, but the question was only about explicitly mentioning None